Makefile

```diff
@@ -33,3 +33,8 @@ test:
 .PHONY: pr-template
 pr-template:
 	. ./scripts/bash/pr_options.sh; pr_template
+
+# Compiles connector generator CLI. For more information locate README file near main.go.
+.PHONY: connector-gen
+connector-gen:
+	go build -o ./bin/cgen ./scripts/connectorgen/main.go && echo "now run command: ./bin/cgen"
```

Comment:
It only `builds` an executable. It doesn't run it, only tells where it is located and how it is called to run later.
As a modification it can be installed so that the command will be available under `$PATH`.